client/js/components/RouteTrips/index.js

```diff
@@ -0,0 +1,64 @@
+import React, {Component, PropTypes} from 'react';
+import {connect} from 'react-redux';
+import {createSelector} from 'reselect';
+
+class RouteTrips extends Component {
+  render() {
+    const trips = this.props.trips.map((trip, i) => {
+      return <li key={i}>{trip.tripHeadsign}</li>
+    })
+    return (
+      <div className='upcomingTrips'>
+        <h2>Upcoming Trips</h2>
+        <ol className='upcomingTripsList'>
+          {trips}
+        </ol>
+      </div>
+    );
+  }
+}
+
+RouteTrips.propTypes = {
+  routeId: PropTypes.string.isRequired,
+  // routes: PropTypes.arrayOf(PropTypes.object).isRequired,
+  // stops: PropTypes.arrayOf(PropTypes.object).isRequired,
+};
+
+const tripsDetailsSelector = createSelector(
+  (state, props) => state.data.routes[props.routeId],
+  (state) => state.data.tripsDetailsForRoute,
+  (route, TripsDetailsForRoute) => {
```

Comment:
This should be lowercase, `tripsDetailsForRoute`. I know this is copypasta, but just fyi.